docs/Learning-Environment-Examples.md

```diff
@@ -31,9 +31,9 @@ If you would like to contribute environments, please see our
     * Visual Observations: 0
 * Reset Parameters: None
 
-## 3DBall
+## 3DBall: 3D Balance Ball
```

Comment:
Is that redundant ?

Reply:
3DBall is the name of the Unity scene, so I did not want to remove that reference.

Reply:
Should we maybe rename the scene then?

Reply:
Up to you folks :). I think its ok for this one example scene to have a scene name and human-readable reference since it serves as our getting started example.

Reply:
For v0.3 I think it is fine as-is.